CONTRIBUTING.md

```diff
@@ -50,17 +50,84 @@ type checking. Any code changes that you contribute should pass these checks.
 The easiest way to get set up for development is to run the following script:
 
 ```bash
+# first fork the repo on github
+
+#then clone your fork
+git clone https://github.com/<your-github-username>/gluon-ts.git
+
+# check if condo is up to date
+Conda update conda
+
+# create virtual env (can change python version if you use newer/older
+# see if any breaking changes
+conda create -n gluon-ts python=3.6 anaconda
+
+# after aeons of dependencies installing
+conda activate gluon-ts
+
+#confirm python version
+Python -V
+
+#enter the cloned repo
 ./dev_setup.sh
+
+#NOTE: you may get some setup lines that are red in stdout
+#In my case it looks like those were ok, on re-running ./dev_setup.sh had no red #stdout lines
+
+# now setup the main repo as upstream remote
+git remote add upstream https://github.com/awslabs/gluon-ts.git
```

Comment:
If I’m not mistaken, with this setup `master` keeps tracking your fork’s `master`. I would avoid putting instructions, and maybe just link to [this](https://help.github.com/en/articles/working-with-forks).

Reply:
What I've done is included here the description from github:
https://help.github.com/en/articles/configuring-a-remote-for-a-fork
which is 2 links (1 more link past your link), just trying to make it easier for external users. IMO the lower overhead for contribution the better. Will remove and include the link you recommend.